docs/advanced-entity-analytics/api/asset-criticality-api-bulk-upsert.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[asset-criticality-api-bulk-upsert]]
+=== Bulk Upsert Asset Criticality Record
+
+Create or update multiple Asset Criticality Records.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/asset_criticality/bulk`
+
+==== Request body
+
+A JSON object defining the Asset Criticality Records.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+|`records` |Array |Array of records, cannot be longer than 1000 records.
+|Yes
+|`records[].id_field` |String |Which field contains the entity ID, must be either `user.name` or `host.name`.
```

Comment:
I dont know how best to format arrays of objects in this table? I have gone for a[].b but I am sure this is wrong